apis/v1/controllers/addresses.controller.js

```diff
@@ -27,6 +28,17 @@ exports.getAllAddresses = async (req, res, next) => {
         const addresses = await Address.find()
         .skip(skip) //call the skip variable
         .limit(limit); // sets the limit on the number of results to return
+
+        // addresses.forEach(address => address.status = address.status.toString())
+
+        let status = req.query.status; //
+        status = (status) ? status.toUpperCase() : undefined;
+        status = (statusMap.has(status)) ? statusMap.get(status) : undefined;
+
+        const searchQuery = {
+            parishName : req.query.parishName,
+            status: status
+        }
```

Comment:
There is no search being done on the addresses. As far as I can see. it finds all addresses then performs the skip and limit the results in addresses. However no filter is being done. the searchQuery is not being used.